pkg/operator/controller/controller_dns_configmap.go

```diff
@@ -31,7 +31,10 @@ var corefileTemplate = template.Must(template.New("Corefile").Parse(`{{range .Se
 {{end -}}
 .:5353 {
     errors
-    health
+    health {
+        lameduck 60s
```

Comment:
I'm not trying to suggest that `60s` is an unacceptable lameduck period, but I'm curious to hear how you arrived at this value.

Reply:
The readiness probe has a period of 10 seconds and a failure threshold  of 3, so I figure it might take 30 seconds for the kubelet to recognize that the pod is unresponsive, and I doubled that for good measure.